src/frontend/src/molecules/PostCard.jsx

```diff
@@ -26,10 +26,20 @@ const PostCard = ({ post }) => {
     }
   }
 
+  function toggleFavorite(e) {
+    e.preventDefault()
+    const heart = e.target
+    if (heart.className === "far fa-heart") {
+      heart.className = "fas fa-heart"
+    } else if (heart.className === "fas fa-heart") {
+      heart.className = "far fa-heart"
+    }
```

Comment:
thanks for the contribution @CrappyDill !! great job, this is conceptually great!

quick note for react: in react, application state drives the DOM.  one never modifies the DOM directly. we actually want to use the `useState` hook in this case.

I'll accept this in and then refactor it, then share the new code with you :-)

Reply:
Here you go: https://discord.com/channels/@me/846852959886901308/849697494761078795